### PR TITLE
Adds Cowboy Repeaters and Brush Guns

### PR DESCRIPTION
**Pull request by JohnOxford**
Adds Cowboy Repeaters, chambered in .44 Magnum, and Brush Guns, chambered in 45-70. Patrol Rangers were given the brush gun and i'll see about making the repeaters spawn in the legion armory.

## Screenshots (if appropriate):
072170d659dc216519e81e1c441a6dbe
28c7560a85666ca06ce48e7c346771d4
## Changelog (neccesary)
:cl:
add: Added cowboy repeaters.
add: Added brush guns.
add: Added brush guns to the Patrol Ranger's inventory.
/:cl:

(this has been tested)

### CODE REVIEW

**Comment by RenGusta** [comment]
handfuls of bullets when

File: code/modules/jobs/job_types/ncr.dm [hunk 1]
```diff
@@ -367,7 +367,6 @@ Veteran Ranger
 	suit = 			/obj/item/clothing/suit/armor/f13/rangercombat
 	head = 			/obj/item/clothing/head/helmet/f13/ncr/rangercombat
 	gloves =		/obj/item/clothing/gloves/fingerless
-	suit_store = 	/obj/item/gun/ballistic/shotgun/automatic/antimateriel
```
Comment (Light-Weave) [inline code/modules/jobs/job_types/ncr.dm]: 1) No.
2) Fuck no. 
3) Detail this in the PR.
4) You're leaving VRs without a primary weapon.

Comment (Light-Weave) [reply code/modules/jobs/job_types/ncr.dm]: Also make 45LC craftable in the lathe if you're giving 4570s to anyone but the VR.

**Comment by companoo** [comment]
if your going to take veteran rangers anti material don't hide it please

**Comment by JohnOxford** [comment]
honest mistake, i was trying to port the camp follower thing that allows you to pick different jobs and doing so required removing the original spawn of the AMR, i forgot to add it back when i decided against giving the option to veteran rangers and just giving it to normal rangers

the intent of the test is the functionality of the weapons and them spawning on normal rangers, my apologies.

**Comment by Light-Weave** [comment]
nah, u good

I'm down with giving patrangers brush guns, but they need a way to make more ammo from the lathe- .45 LC preferably.

**Comment by Light-Weave** [comment]
.44 is 35 damage 0 AP
.45-70 is 60 damage -20 AP and uncraftable
.45 LC is 45 damage 20 AP

**Comment by GhostEcho** [comment]
I want Ren's input on several decisions such as giving it to Patrol rangers

**Pull request merged**